app/controllers/api/todos_controller.rb

```diff
@@ -10,27 +10,26 @@ def index
     render "index.json.jb"
   end
 
-  def edit
-    @todo = Todo.find(params[:id])
+  def update
+    @todo = Todo.find_by(id: params[:id])
+    if @todo.update(todo_params)
+      @todo.save
+    end
```

Comment:
```suggestion
   todo = Todo.find_by(id: params[:id])
   render json: { status: 200 } if todo.update(todo_params)
   render json: { status: 500 } 
Write
```

- こちらもjbを使ってないのでグローバルに公開する必要はなさそうですね

- .saveはupdateされているので必要ないですね！

- errorが発生した場合には500を返すようにした方がいいですね

- 基本的にrails の作法としてif分は１行でかけるなら後ろ側にif分を書きます
またelseも基本的には使わない形で条件判するのが作法なので上記のようにする方が見やすそうです。